app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="text_main_activity_margin_start">15dp</dimen>
+    <dimen name="text_main_activity_margin_end">15dp</dimen>
+    <dimen name="text_main_activity_margin_top">20dp</dimen>
+    <dimen name="text_main_activity_text_size">35sp</dimen>
+
+</resources>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
done